### PR TITLE
fix 1519808. Only annotate ops projects when openshift_logging_use_op…

### PR DESCRIPTION
**Pull request by jcantrill**
…s=true

### CODE REVIEW

File: roles/openshift_logging/tasks/delete_logging.yaml [hunk 1]
```diff
@@ -107,6 +107,25 @@
     - logging-fluentd
     - logging-mux
 
+# remove annotations added by logging
+- oc_obj:
```
Comment (ewolinetz) [inline roles/openshift_logging/tasks/delete_logging.yaml]: is this step necessary if we already have the names of the ops projects?

File: roles/openshift_logging/tasks/delete_logging.yaml [hunk 2]
```diff
@@ -107,6 +107,25 @@
...
+  register: __logging_ops_projects
+
+- name: Remove Annotation of Operations Projects
+  command: >
+    {{ openshift.common.client_binary }}
+    --config={{ openshift.common.config_base }}/master/admin.kubeconfig
+    annotate ns
+    {{ item.item }}
+    openshift.io/logging.ui.hostname-
+  with_items: "{{ __logging_ops_projects.results }}"
```
Comment (ewolinetz) [inline roles/openshift_logging/tasks/delete_logging.yaml]: can you use a loop variable in here? http://docs.ansible.com/ansible/latest/playbooks_loops.html#loop-control

File: roles/openshift_logging/tasks/delete_logging.yaml (hunk 1 above)
Comment (jcantrill) [reply roles/openshift_logging/tasks/delete_logging.yaml]: I believe, similar to the annotation task, we list the projects and get back those that actually exist in context of those we  expect to be there.  We then use that result to further remove the annotation.  We can remove this step if you feel removing the annotation and ignoring failure should not actual exist is acceptable

**Pull request merged**